action.yml

```diff
@@ -13,7 +13,15 @@ inputs:
     description: >-
       Array of products to install
     required: false
-    default: MATLAB
+    default: |
+      MATLAB
+      Parallel_Computing_Toolbox
+  location:
+    description: >-
+      Destination location for MATLAB install
+    required: false
+    default: /opt/matlab
+
 runs:
-  using: node12
+  using: node16
```

Comment:
We show `default: R2021b` up on line 13. Shouldn't that be `default: latest`?

Reply:
Fixed